### DISCUSSION TITLE
async stateless processors

### DISCUSSION THREAD

**Discussion by rukawa917** [LGouellec/streamiz#207] [Q&A] [answered]
Are async stateless processors like ForeachAsync available in the current NuGet package? If so, would it be possible for some simple example if anyone has used it? I can only find non-async processors.

**Comment by LGouellec** [comment, marked as answer]
1.4.0 will contain this feature. It will be available next week.

For now, you can use the last release candidate 1.4.0-RC5.

Documentation available here : https://lgouellec.github.io/kafka-streams-dotnet/async-processing.html
